test/test_ltc2387.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+hardware = "ltc2387"
+classname = "adi.ltc2387"
+
+#########################################
+@pytest.mark.iio_hardware(hardware)
```

Comment:
Should be @pytest.mark.iio_hardware(hardware, True) otherwise [emulation support will turn on](https://pytest-libiio.readthedocs.io/en/latest/emulation/)